src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,104 @@
-export const Pagination = () => {};
+import { MouseEvent } from 'react';
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage?: number;
```

Comment:
I see you're passing currentPage anyway, so no needs to make this prop optional

Reply:
Yes, but this way the Pagination component will be less versatile for another case. 
Well, that was the requirement in the task)

Reply:
Agree. Haven't noticed this requirement
```
currentPage={1} /* optional with 1 by default */
```